setup.py

```diff
@@ -4,8 +4,22 @@
 """The setup script."""
 
 from setuptools import find_packages, setup
+from setuptools.command.build_py import build_py
+from pathlib import Path
 from typing import Dict, List
 
+
+class BuildPyCommand(build_py):
+    """Check for existence of XSLT before building."""
+    def run(self):
```

Comment:
Does `super().run()` need to be called as well?

Reply:
`build_py.run(self)` below calls the default run behavior after this check happens.

Reply:
🤦‍♀️ whoops, should have noticed that. Thanks!